recipes/declare/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "declare" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b4fea4732e30bfeab4cb8fea856918d718f85f6481cee9dd0c4fc6738949b66c
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - poetry-core
+  run:
+    - python >=3.8
+    - typing-extensions >=4.10.0,<5.0.0
+
+test:
```

Comment:
Could you try adding a `pip check` test?

https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L61-L64

Reply:
Thanks, done.